.gitignore

```diff
@@ -158,3 +158,7 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+notebooks/sg_argosim_notebook.ipynb
```

Comment:
This should not be included to the pull request, I think it would be specific to your fork.

Reply:
Sorry, it was a stupid mistake, I did not mean to include it.